frontend/src/components/Footer/StyledFooter.ts

```diff
@@ -32,63 +32,63 @@ const StyledFooter = styled.footer`
     }
   }
 
-  #logo {
+  #footer__logo {
     margin-left: 25px;
   }
 
-  .center-links {
+  .footer__center-links {
     display: flex;
     justify-content: center;
     flex-grow: 1;
   }
 
-  .icons {
+  .footer__icons {
     display: flex;
     align-items: center;
     justify-content: flex-end;
     height: 96px;
     line-height: 0;
   }
 
-  .icons img {
+  .footer__icons img {
     width: 32px;
     height: 32px;
   }
 
-  .icons span {
+  .footer__icons span {
     margin-right: 5px;
   }
 
   @media (max-width: 639px) {
-    .footerlinks {
+    .footer__links {
       height: 257px;
       padding: 32px 0 0 16px;
     }
 
-    .footerlinks,
-    .footerlinks div.center-links {
+    .footer__links,
+    .footer__links .footer__center-links {
```

Comment:
Here seems that .footer__center-links is a modifier and by BEM we have footer__link--center (name--value).